pom.xml

```diff
@@ -93,8 +93,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.3</version>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>${maven.compiler.source}</source>
+                    <target>${maven.compiler.target}</target>
```

Comment:
This value would be 8 instead of 1.8, I guess that's fine?

Reply:
It's fine to tell Maven that uses Java 8 adding source and target 1.8.